src/main/java/model/LottoRank.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public enum LottoRank {
+    FIFTH(3, new Money(5000)),
+    FOURTH(4, new Money(50000)),
+    THIRD(5, new Money(1500000)),
+    SECOND(5, new Money(30000000)),
+    FIRST(6, new Money(2000000000));
+
+    private final int numberOfMatches;
+    private final Money prize;
+
+    LottoRank(int numberOfMatches, Money prize) {
+        this.numberOfMatches = numberOfMatches;
+        this.prize = prize;
+    }
+
+    public static Optional<LottoRank> get(int numberOfMatches, boolean containsBonusNumber) {
+        if (numberOfMatches == SECOND.numberOfMatches && containsBonusNumber) {
```

Comment:
```suggestion
        if ((numberOfMatches == SECOND.numberOfMatches) && containsBonusNumber) {
```
연산자 우선순위가 헷갈릴 수 있으니 괄호로 한 번 더 감싸줘서 가독성을 높여주세요